source/unified-test-format/tests/Makefile

```diff
@@ -1,8 +1,8 @@
-SCHEMA=../schema-1.4.json
+SCHEMA=../schema-1.5.json
 
-.PHONY: all invalid valid-fail valid-pass versioned-api load-balancers gridfs transactions crud collection-management HAS_AJV
+.PHONY: all invalid valid-fail valid-pass versioned-api load-balancers gridfs transactions crud collection-management sessions command-monitoring HAS_AJV
 
-all: invalid valid-fail valid-pass versioned-api load-balancers gridfs transactions crud collection-management
+all: invalid valid-fail valid-pass versioned-api load-balancers gridfs transactions crud collection-management sessions command-monitoring
```

Comment:
Re: "sessions", I assume that was missed in https://github.com/mongodb/specifications/pull/1022?

cc: @BorisDog

Reply:
Boris is OOO but yes we forgot to add it in the snapshot sessions PR.

Reply:
That is correct, I figured I'd sneak it in there.